contrib/node/src/python/pants/contrib/node/subsystems/package_managers.py

```diff
@@ -0,0 +1,231 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+from pants.contrib.node.subsystems.command import command_gen
+
+
+LOG = logging.getLogger(__name__)
+
+PACKAGE_MANAGER_NPM = 'npm'
+PACKAGE_MANAGER_YARNPKG = 'yarnpkg'
+PACKAGE_MANAGER_YARNPKG_ALIAS = 'yarn'
+VALID_PACKAGE_MANAGERS = [PACKAGE_MANAGER_NPM, PACKAGE_MANAGER_YARNPKG, PACKAGE_MANAGER_YARNPKG_ALIAS]
+
+
+# TODO: Change to enum type when migrated to Python 3.4+
```

Comment:
This is a useful comment, I'm not familiar enough with python 3 to know where all the migration points are so this is helpful.